examples/CustomAssignmentConsoleApp/appsettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "FeatureManagement": {
+    "DailyDiscount": {
+      "Assigner": "DayOfWeek",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "Default",
+          "ConfigurationReference": "DailyDiscount:Default"
+        },
+        {
+          "Name": "Default",
```

Comment:
>Default [](http://example.com/codeflow?start=19&length=7)

Tuesday?

Reply:
Yes, shouldn't be "Default" at the least, thanks.